proxy/coco/src/peer.rs

```diff
@@ -261,6 +274,24 @@ impl Api {
         Ok(project_meta)
     }
 
+    /// Query the network for providers of the given [`RadUrn`].
+    ///
+    /// Notes:
```

Comment:
```suggestion
    /// NB(nuno):
```

Let's treat these nota benes like TODOs, FIXMED, etc.

Reply:
I'd disagree that these are TODOs. The below are semantics that a user of this function should be aware of. So they're acting as documentation that the user of this function should be aware of.

Reply:
Is that the same thing? He's documenting what callers should be aware of.

Reply:
> I'd disagree that these are TODOs. The below are semantics that a user of this function should be aware of. So they're acting as documentation that the user of this function should be aware of.

I did not say that these are TODOs. My point is that we should keep the same notation style, where `<TOKEN>(<author>): <Comment>`.

Reply:
Ah, I see. I got confused by your wording. I find the author being in documentation strange.

If you wanted to track them down just `git blame` :sweat_smile:

Reply:
exactly, these are documentation notes, not notes for me.

Reply:
Having the author in there helps contextualising and potentially knowing who to ask when advise is needed. Git blame can fall apart esp. when moving code and on large refactors.

Reply:
Sorry, I can't see the point. This is documenting the caveats of the function, there is no need to know whom to contact. I sense that if I had inlined these notes that this wouldn't be a discussion anymore?

Reply:
Not planning to die on this hill. :japanese_ogre:

Reply:
I just made the decision for you, so feel free to replace the commentary with "RTFM": radicle-dev/radicle-link#314